Add IPFS Podcasting "App"

Cameron-IPFSPodcasting opened the pull request:
Does IPFS Podcasting qualify as an "app"? : ) 

Always worry I didn't do the PR correctly. 

Added a "Start9" platform to my entry, but didn't touch the schema.

ericpp commented:
Hi Cameron! You do need to need to add new platforms to the schema before you can use them. You would add Start9 to the list of platforms in the `apps-schema.json` file here: https://github.com/Podcastindex-org/web-ui/blob/a9bed25f9a277359d82eb5adbf8d8d0352e983c2/server/data/apps-schema.json#L56-L83

ericpp commented on server/data/apps.json:
```diff
@@ -2804,5 +2804,40 @@
+  {
+    "appName": "IPFS Podcasting",
+    "appType": [
+      "hosting",
```
The `directory` type might also apply since you are hosting a directory of IPFS-ified podcast feeds and episodes.

```suggestion
      "directory",
      "hosting",
```

daveajones commented:
Got it!

Cameron-IPFSPodcasting commented:
I added Start9 to the platforms and fixed "tagLine" (from "tagline"), then was unsure how to proceed.

Looks like he completed this as I was typing (thanks Dave!).

daveajones commented:
@ericpp Are you on podcastindex.social?

ericpp commented:
@daveajones No, but I could join if it would be helpful. I generally try to avoid social media as much as possible.

daveajones commented:
No worries.  Just wondering.
